.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,183 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  build:
+    name: Building on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    env:
+      TOKEN: ${{ secrets.SOFA_REPO_READ_TOKEN }}
+      SOFA_ROOT: /opt/sofa
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+        with:
+          repository:  jnbrunet/plugin.SofaPython3
+          ref: install_tests
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

Comment:
Is this where do you download the dev pacakges for libpython3.7? I'm not sure I remember how those actions are working and where they're located

Reply:
This action is used because many python versions are pre-installed on the runners. It remove us the burden of setting cmake python's path and runtime python's path manually. We could even use this to pass different python versions in the CI matrix (3.6, 3.7, 3.8 for example). In fact, it would be awesome to have a matrix like:

```yaml
strategy:
    matrix:
        os: [ubuntu-latest,  macos-latest, windows-latest]
        python: [3.6, 3.7, 3.8]
        pybind11: [2.4, 2.5]
```

Especially given that with ccache hits, each build takes about 2-3 min, which means the complete matrix wouldn't take too much time to build and test.